### ISSUE TITLE
sorry can I ask some questions about east training here ?

### ISSUE THREAD

**Issue by blankWorld** [zxytim/fast-gmm#2]
hi, have you tried pvanet as basenetwork? I tried pvanet using caffe but encountered overfitting problem.
my training sets is 950 images from icdar 2015 trainningsets( the other 50 images as validation sets) and 229 images from icdar 2013.
model is trained by online data augmentation which includes scaling and rotations between ±30°. iou loss overfits a lot that when trainning iou descend to 0.25 validation iou loss still stays high at 0.7. I think I have confirmed everything so much that I can not solve this problem. please help me, Mr. tim!!!!!!. I have cost two month on this problem,  all my thanks for you!!

**Comment by zxytim** [comment]
Hi World, you'd better submit the issue at https://github.com/argman/EAST but
not this in this "fast-gmm" repo.
Nonetheless, you can try playing around with weight decay, e.g., add 1e-6
weight decay while training. If not working, use more weight decay, like
1e-5.

**Comment by blankWorld** [comment]
OK， 3q~~ I will try more weight decay in my training and ..... last question,   do you use 'not care' texts in your training

**Comment by zxytim** [comment]
No. The loss regarding "do not care" areas are muted.

BTW, only Chinese can understand "3q"

**Comment by blankWorld** [comment]
haha yes 看了你的文章 感触颇多 这么简单的pipeline也能有如此的效果 准备稍微深入一下 顺便 祝你事业顺利
